### ISSUE TITLE
Hamburger menu at 200% zoom

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#241]
## Location: Across the entire site
At 200% zoom in the browser, when I keyboard tab through the hamburger menu, I am not able to move focus to the "Blog" or "About Us" menu items. I am also unable to use my mouse to scroll down to see those last 2 menu items.

## Definition of Done
- I should be able to keyboard tab to each menu item without them being obscured.
- I should be able to use my mouse to scroll down to view all menu items.

**Comment by mejiaj** [comment]
There's a few issues with the current implementation.

Mainly:
- We're not using SVG's for the icons, so scaling is a little extra work.
- Using `position: sticky` on `header.bix-site-header` limits the height of the nav. You _could_ get around it with viewport units, but it'll be difficult to get it right on all sizes.


There's a few possible ways we can address this:

1. **Separate, duplicate, mobile nav.** Toggle visibility depending on the view.
2. **Make the header full width.** This would allow you to position the navigation statically and make it full-width without CSS.
3. **Make the mobile dropdown cover entire screen.** Similar to how USWDS makes the menu cover the entire screen, [example](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?p=all&args=&id=pages-documentation-page--documentation-page&viewMode=story) — zoom into 400%.

The lowest LOE is probably item 1.
